### PR TITLE
Organization Page Styling

### LINKED ISSUE #48: Improve Organization Page Styling
# Improve Organization Page Styling
Make minor refinements to the preexisting styling to both clean up the CSS file and fix issues such as the line height and inconsistent margins regarding the organization cards.

## Why?
This is important to improving the overall appearance of the website for users to ensure that the site looks clean and professional.

## Acceptance Criteria
- [x] Set line height to two times the font size plus a little extra for the space between each line.
- [x] Fix the bottom margin of the card so the spacing is consistent with the top part of the card.
- [x] Reduce the font size on the Organization Details page and restyle the main card accordingly.
- [x] Align the organization logo to the top left of the card.

### PR DESCRIPTION
**Pull request by briannata**
This pull request edits the CSS and styling of the organizations and organization details pages.

## Major Changes
- Set line height to two times the font size plus a little extra for the space between each line
 - Fix the bottom margin of the card so the spacing is consistent with the top part of the card
 - Reduce the font size on the Organization Details page and restyle the main card accordingly
 - Align the organization logo to the top left of the card
 - Response styling on the organization details page (for phones, tablets, laptops, and monitors)

## Testing
These changes were thoroughly tested on the local server by running `honcho start` and with various screen sizes.

Original Laptop View

Monitor View

Tablet Views (Portrait and Landscape)

Phone Views (Portrait and Landscape)
 

Please provide feedback on some of the styling decisions in the comments below. I would also appreciate any feedback on the CSS and styling since I only tested this on my personal laptop and don't know what it'll look like on other screens.

> *This PR resolves #48*

### CODE REVIEW

**Comment by briannata** [comment]
When aligning the logo with the top of the organization details header, I was wondering whether I should add a little padding at the top since the organization title has some padding at the top as well.

Right now:
image

If I uncommented my padding-top line in the CSS:
image

Could also make the padding-top in the second picture less drastic (such as 0.5rem instead of 0.75rem).

**Comment by briannata** [comment]
On the organization details page, when the width of the screen is small enough, the organization logo jumps to the top instead of the side, looking like this:
image

I would prefer to have it aligned with the organization name and social media icons, but the only way I could think of is making another organization logo  that only shows when the width is small enough. This is because the organization logo is in a div that separates it from the rest of the content. The organization name, social media icons, and description are all in a separate div together so I would have to insert the mobile only organization logo into that div to align it with the organization name and social media icons properly.

TLDR: Is making an organization logo that shows on mobile only acceptable if it helps make the organization details page look better on less wide screens?

**Comment by atoneyd** [comment]
I think you accidentally made this pull request to merge into main, rather than the organization-merge branch; could you edit that? And yeah I agree I think the move for mobile would be to have the logo aligned with the title. I also like the second option where the top of the logo is aligned with the top of the text. Thanks for considering all these options!

**Comment by jadekeegan** [comment]
The changes look great overall! One thing though is that when you resize the window on laptop for the Organization Detail pages, an extra left margin is added.
image

**Comment by briannata** [comment]
> One thing though is that when you resize the window on laptop for the Organization Detail pages, an extra left margin is added and the mobile icons overflow outside the container.

I couldn't recreate the icons floating off the mat card, but I fixed the margin! Ready for another review.
image

**Comment by jadekeegan** [comment]
The styling issues don't appear in the dev tools when I change the window sizing in "responsive mode", but when I minimize the browser window so it isn't full screen, the issue appears.
image

**Review by jadekeegan — CHANGES_REQUESTED** [review]
Everything looks good in the dev tools when I change the window sizing in "responsive mode", but when I minimize the browser window so it isn't full screen, the same issue appears again.

**Review by jadekeegan — APPROVED** [review]
LGTM! Ready to merge! Thank you for making those changes! :)
